stl/src/vector_algorithms.cpp

```diff
@@ -1327,6 +1399,144 @@ namespace {
         }
     }
 
+    // _Minmax has exactly the same signature as the extern "C" functions
+    // (__std_min_Nn, __std_max_Nn, __std_minmax_Nn), up to calling convention.
+    // This makes sure the template specialization is fused with the extern "C" function.
+    // In optimized builds it avoids an extra call, as this function is too large to inline.
+    template <_Min_max_mode _Mode, class _Traits, const bool _Sign>
+    auto __stdcall _Minmax(const void* _First, const void* const _Last) noexcept {
+        using _Ty = std::conditional_t<_Sign, typename _Traits::_Signed_t, typename _Traits::_Unsigned_t>;
+
+        _Ty _Cur_min_val; // initialized in both of branches below
+        _Ty _Cur_max_val; // initialized in both of branches below
+
+        // We don't have unsigned 64-bit stuff, so will use sign correction just for that case
+        constexpr bool _Sign_correction = sizeof(_Ty) == 8 && !_Sign;
+
+#ifndef _M_ARM64EC
+        if (_Byte_length(_First, _Last) >= 16 && _Traits::_Sse_plain_min_max_available()) {
+            const size_t _Sse_byte_size = _Byte_length(_First, _Last) & ~size_t{0xF};
+
+            const void* _Stop_at = _First;
+            _Advance_bytes(_Stop_at, _Sse_byte_size);
+
+            auto _Cur_vals = _Traits::_Load(_First);
+
+            if constexpr (_Sign_correction) {
+                _Cur_vals = _Traits::_Sign_correction(_Traits::_Load(_First), false);
+            }
+
+            auto _Cur_vals_min = _Cur_vals; // vector of vertical minimum values
+            auto _Cur_vals_max = _Cur_vals; // vector of vertical maximum values
+
+            for (;;) {
+                _Advance_bytes(_First, 16);
+
+                if (_First == _Stop_at) {
+                    // Reached end.
+                    // Compute horizontal min and/or max. Determine horizontal and vertical position of it.

```

Comment:
Question: This was copy-pasted, but "Determine horizontal and vertical position of it." should have been dropped, right?

https://github.com/microsoft/STL/blob/192a84008a59ac4d2e55681e1ffac73535788674/stl/src/vector_algorithms.cpp#L1152

Reply:
Right.